src/validator.js

```diff
@@ -368,6 +383,11 @@
     Extended.prototype = Object.create( Assert.prototype );
     Extended.prototype.constructor = Extended;
 
+    // Copy all the static methods.
+    Object.keys( Assert ).forEach( function( key ) {
```

Comment:
I have no idea which browsers are supported by the current version. Is there anything you can point me to in terms of compatibility tests we're doing?